app/models/validator.rb

```diff
@@ -81,6 +81,7 @@ class Validator < ApplicationRecord
 
   delegate :data_center_key, to: :data_center_host, prefix: :dch, allow_nil: true
   delegate :address, to: :validator_ip_active, prefix: :vip, allow_nil: true
+  delegate :commission, to: :validator_score_v1, allow_nil: true
```

Comment:
Commission is score's attribute, so there's no point of delegating it to score model.
Check score method content in validator model.